telco-hub/configuration/reference-crs-kube-compare/optional/logging/clusterLogForwarder.yaml

```diff
@@ -0,0 +1,50 @@
+---
+# ClusterLogForwarder for Telco Hub
+# Forwards audit and infrastructure logs to Kafka with hub-specific labeling
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+  annotations:
+    # Deploy after cluster logging operator is ready
+    argocd.argoproj.io/sync-wave: "10"
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    # Ignore controller-managed status differences in ArgoCD
+    argocd.argoproj.io/compare-options: IgnoreExtraneous
+spec:
+  # Filters add metadata labels to log records for identification
+  filters:
+  - name: telco-hub-labels
+    type: openshiftLabels
+    # Add these labels to all forwarded log records
+    openshiftLabels:
+      cluster-role: hub                # Identifies this as hub cluster logs
+      environment: production          # Environment designation
+      telco-component: management      # Component categorization
```

Comment:
This content can be entirely up to the user. We won't require any specific labels.